src/main/java/sirius/biz/storage/layer2/BlobDispatcher.java

```diff
@@ -181,12 +181,26 @@ public DispatchDecision dispatch(WebContext request) throws Exception {
         Values uriParts = Values.of(uri.split("/"));
         String type = uriParts.at(0).asString();
         if (Strings.areEqual(type, PHYSICAL_DELIVERY) && uriParts.length() == 5) {
-            String filename = stripAdditionalText(uriParts.at(4).asString());
+            String physicalKey = stripAdditionalText(uriParts.at(4).asString());
```

Comment:
Would it perhaps be possible to eliminate the MagicNumbers? Wherever uriParts.at() is used a descriptive constant might be helpful.

Reply:
Would appreciate it if you create the constants as you described. Only the ones that are really used should probably be enough or?

Reply:
I agree